### PR TITLE
WIP:RHOARODOC-1582: Launcher health check, REST API, and Relational Database capabilities

### PR DESCRIPTION
**Pull request by huntley**

### CODE REVIEW

File: docs/topics/assembly_launcher-capabilities.adoc [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+:parent-context: {context}
+
+When you create a new application using link:{link-launcher-oso}[{name-launcher-oso}, you can add the basic building blocks of a cloud-based application. The {name-launcher-oso} refers to these building blocks as capabilities. An application you create using {name-launcher-oso} automatically includes a health check capability. In addition, you can add relational persistence to integrate with a relational database and HTTP API capabilities to your application.
+
+// The health check capability allows you to monitor the health of your application, which includes whether an application is available and whether it is able to service requests. In addition to a health check, you can also add a REST API or a relational database backend capability to your application.
+//
+
+include::con_launcher-health-check-capability.adoc[leveloffset=+1]
+:parent-context: {context}
```
Comment (ncbaratta) [inline docs/topics/assembly_launcher-capabilities.adoc]:
```suggestion
:context: {parent-context}
```

File: docs/topics/con_launcher-health-check-capability.adoc [hunk 2]
```diff
@@ -0,0 +1,12 @@
+[id="{name-launcher-oso}-health-check-capability_{context}"]
+= {name-launcher-oso} Health Check Capability
```
Comment (ncbaratta) [inline docs/topics/con_launcher-health-check-capability.adoc]: Sentence case

File: docs/topics/assembly_launcher-capabilities.adoc [hunk 3]
```diff
@@ -0,0 +1,12 @@
+[id='launcher-capabilities.adoc_{context}']
+= Launcher Capabilities
```
Comment (ncbaratta) [inline docs/topics/assembly_launcher-capabilities.adoc]: Sentence case

**Comment by tradej** [comment]
ok to test

File: docs/topics/assembly_health-check-example-application.adoc [hunk 4]
```diff
@@ -9,20 +9,16 @@
 [id='example-health-check-{parameter-runtime-anchor}']
 = {name-example-health-check} example for {runtime}
 
-include::health-check-example-intro-paragraph.adoc[leveloffset=+1]
+Example proficiency level: 
 
-NOTE: If you are not familiar with the health check terminology, see the xref:health-check-concepts_{context}[] section first.
+ifdef::docs-topic[xref:proficiency_foundational[*{proficiency-foundational}*].]
```
Comment (tradej) [inline docs/topics/assembly_health-check-example-application.adoc]: See https://github.com/fabric8-launcher/launcher-documentation/pull/1308/files#r315166040

File: docs/topics/assembly_health-check-example-application.adoc [hunk 5]
```diff
@@ -9,20 +9,16 @@
...
+ifndef::docs-topic[link:https://launcher.fabric8.io/docs/thorntail-runtime.html#proficiency_levels[*{proficiency-foundational}*^].]
+
+When you deploy an application, it is important to know if it is available and if it can start handling incoming requests. Implementing the _health check_ pattern allows you to monitor the health of an application, which includes if an application is available and whether it is able to service requests.
```
Comment (tradej) [inline docs/topics/assembly_health-check-example-application.adoc]: One sentence per line, please.

File: docs/topics/assembly_launcher-capabilities.adoc [hunk 6]
```diff
@@ -0,0 +1,11 @@
+[id='launcher-capabilities.adoc_{context}']
+= Launcher capabilities
+
+:parent-context: {context}
+
+When you create a new application using link:{link-launcher-oso}[{name-launcher-oso}], you can add the basic building blocks for cloud-based applications. The link:{link-launcher-oso}[{name-launcher-oso}] refers to these building blocks as capabilities. Available capabilities include a health check, a REST API, and a relational database backend.
```
Comment (tradej) [inline docs/topics/assembly_launcher-capabilities.adoc]: The last sentence should be a bullet list.

File: docs/topics/ref_health-check-resources.adoc [hunk 7]
```diff
@@ -1,16 +1,16 @@
...
 
-More background and related information on health checking can be found here:
+More background and related information on health checks can be found here:
 
 * link:https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html/developer_guide/dev-guide-application-health[Application Health in OpenShift]
 
 * link:https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/[Kubernetes Liveness and Readiness Probes]
 
-ifndef::built-for-spring-boot[* link:{link-example-health-check-spring-boot}[{name-example-health-check} for {SpringBoot}]]
+ifndef::built-for-spring-boot,built-for-getting-started[* link:{link-example-health-check-spring-boot}[{name-example-health-check} for {SpringBoot}]]
```
Comment (tradej) [inline docs/topics/ref_health-check-resources.adoc]: Why not show the example app links for individual runtimes in the Getting Started Guide?

Comment (huntley) [reply docs/topics/ref_health-check-resources.adoc]: I think the example applications in the runtime guides are different than the capabilities provided by the launcher. While they are based on the same code, they don't have the same features.

File: docs/topics/assembly_health-check-example-application.adoc (hunk 4 above)
Comment (huntley) [reply docs/topics/assembly_health-check-example-application.adoc]: Added my reply to your link above. I am happy to remove them if you think that is the way to go.

**Comment by huntley** [comment]
Updated PR name to reflect a change. All Launcher capabilities will be developed on this branch.

File: docs/topics/ref_health-check-resources.adoc (hunk 7 above)
Comment (tradej) [reply docs/topics/ref_health-check-resources.adoc]: All right. Please verify with Engineering.

**Comment by inoxx03** [comment]
[test]

**Comment by inoxx03** [comment]
This PR can not be completed due to low priority and lack of team resources. It will be closed and  can be reopened later.

**Pull request closed without merging**